frame/fee-market/src/lib.rs

```diff
@@ -246,13 +245,14 @@ pub mod pallet {
 
 			// Increase the locked collateral
 			if new_collateral >= Self::relayer(&who).collateral {
-				let _ = T::RingCurrency::extend_lock(
+				T::RingCurrency::set_lock(
```

Comment:
I need some time to check the change here, please fix ci firstly.

Reply:
If you have already compared the value. Just set a new lock here and you can get rid of this error handling.

Reply:
I see the concrete implementation difference in this.  LGTM, just waiting for the ci.